pkg/interface/src/apps/chat/components/lib/chat-header.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component, Fragment } from "react";
+import { Link } from "react-router-dom";
+
+import { ChatTabBar } from "./chat-tabbar";
+import { SidebarSwitcher } from "../../../../components/SidebarSwitch";
+import { deSig } from "../../../../lib/util";
+
+
+export class ChatHeader extends Component {
```

Comment:
TS would be nice, but not a dealbreaker

Reply:
TS seems excessive for a component with no logic.